library/library/settings.py

```diff
@@ -27,7 +27,7 @@
 SECRET_KEY = os.getenv('DJANGO_SECRET_KEY')
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = False
```

Comment:
Well, do we use debug mode or not ?

Reply:
I asked @mehalyna about this, she said that we don't have to set `DEBUG=True` in commits